packages/spruce-utils/lib/il8n.tsx

```diff
@@ -0,0 +1,35 @@
+/** Returns a string of hours and minutes. Ex: "4hr 30 min" */
+export const friendlyDuration = (seconds: number, lang?: any): string => {
```

Comment:
Can we use moment for this instead?

https://momentjs.com/docs/#/i18n/

Reply:
Unfortunately this library doesn't have what we need for specifying a duration format in different languages :(